pkg/internal/cli/override.go

```diff
@@ -0,0 +1,17 @@
+package cli
+
+import (
+	"os"
+
+	"github.com/spf13/pflag"
+	"sigs.k8s.io/kind/pkg/log"
+)
+
+func OverrideDefaultName(logger log.Logger, fs *pflag.FlagSet) {
+	if !fs.Changed("name") {
+		if name := os.Getenv("KIND_CLUSTER_NAME"); name != "" {
+			fs.Set("name", name)
+			logger.V(1).Infof("using default cluster name as %q due to KIND_CLUSTER_NAME", name)
```

Comment:
Last nit: it's not the default right?
using cluster name %q from KIND_CLUSTER_NAME

Or just drop this all together. I don't think most apps log this sort of thing.

Reply:
done